README.adoc

```diff
@@ -385,6 +385,24 @@ class Transaction < ActiveRecord::Base
 end
 ----
 
+=== Enums [[enums]]
+
+Prefer hash syntax over arrays for `enums`.
+
+[source,ruby]
+----
+class Transaction < ActiveRecord::Base
+  # bad - implicit values - ordering in array matters
```

Comment:
Can't come up with better wording, do you think that "ordering" is sufficient to emphasize that if something is inserted or removed in the middle, things will break, not only when entries are rearranged?

Reply:
Added more details to the rule description.